README.md

```diff
@@ -61,3 +61,43 @@ docker build -t astral:latest .
 ```
 docker run -p 3000:3000 astral:latest
 ```
+# Logging into vault with OIDC
+
+The rails test's configure the OIDC provider, so if the tests pass,
+you can invoke the oidc login as follows:
+
+```
+  export VAULT_ADDR=http://127.0.0.1:8200; vault login -method=oidc
+```
+
+You should do this on your host machine, not in docker.  This will
+allow a browser window to open on your host.  When it does, select
+"username" option with user test/test.  (That is the username/pw
+configured by the rails tests.)
+
+When that succeeds, you should see something like the following in the cli:
+```
+Success! You are now authenticated
+.
+identity_policies    ["test@example.com"]
+.
+.
+```
+
+Note that "identity_policies" includes "test@example.com", which is the
+policy we created for this user.
+
+To make this work smoothly with the browser, you should add the
+following to the /etc/hosts file on your host:
+
+```
+  127.0.0.1	oidc_provider
+```
+
+Finally, if you run "rails test" a second time, it will recreate the
```

Comment:
This could potentially be super annoying, although I think there's a way to exclude certain tests. Perhaps mention it here?

Reply:
I think memoizing might fix this.  I'll take a look.

Reply:
Actually, even if we memoize or exclude certain tests in a particular run, the next time we restart the docker env, the cookie will still be cached in the browser, so I'm not sure there is a work around for this.

Reply:
Maybe the comment doesn't make it clear, but the cookie only needs to be cleared if you try running "vault login -method=oidc" from the command line again.  It doesn't effect "rails test".  Rather "rails test" effects it.

Bottom line, "rails test" runs fine without worrying about the browser cache.  It is only if you want to run vault login from the command line that you have to clear the cookie.  Does that make sense?

Reply:
I've modified the code so the cookie only expires when the docker vault container is restarted, not every time rails test is run.  Is that sufficient for now?